client/src/components/library/LibrarySearch.jsx

```diff
@@ -24,6 +24,7 @@ export default class LibrarySearch extends Component {
           className="search"
           onChange={evt => this.setState({ searchText: evt.target.value })}
           value={this.state.searchText}
+          onKeyDown={o => (o.key === 'Enter' && this.props.onSearch(this.state.searchText.trim()))}
```

Comment:
While I understand that we want a one-line here, why not `evt` for the local variable instead of `o`? What does `o` means? ... `evt` is widely use as variable for _event_, a [`KeyboardEvent`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) in this case?

Reply:
fixed! 👍